Documentation/Books/AQL/Functions/Document.md

```diff
@@ -47,6 +47,34 @@ FOR attributeArray IN attributesPerDocument
         RETURN {attr, count}
 ```
 
+CHECK_DOCUMENT()
+----------------
+
+`CHECK_DOCUMENT(document) → checkResult`
+
+Returns true if *document* is a valid document object, i.e. a document without any
+duplicate attribute names. Will return *false* for any non-objects/non-documents,
+and *true* for any objects/documents without duplicate attribute names.
+
+Please note that this is an internal function for validating database objects and
+is not supposed to be any useful for anything else. The primary use case for this 
+function is to apply it on all documents in a given collection as follows:
```

Comment:
Could use an explanation what this actually does - get all invalid documents and serialize them to a JSON string for debugging purposes?

Should we add a note to the description of JSON_STRINGIFY(), that it does not check for well-formed-ness of the JSON?